.tekton/catalog-build-pipeline.yaml

```
@@ -0,0 +1,380 @@
+apiVersion: tekton.dev/v1
+kind: Pipeline
+metadata:
+  name: catalog-build-pipeline
+spec:
+  description: |
+    This pipeline is ideal for building and verifying [file-based catalogs](https://konflux-ci.dev/docs/end-to-end/building-olm/#building-the-file-based-catalog).
+
+    _Uses `buildah` to create a container image. Its build-time tests are limited to verifying the included catalog and do not scan the image.
+    This pipeline is pushed as a Tekton bundle to [quay.io](https://quay.io/repository/konflux-ci/tekton-catalog/pipeline-fbc-builder?tab=tags)_
+  params:
```

Can we add this pram and add it to init (https://github.com/konflux-ci/build-definitions/blob/4267ec2c593ecb8cb5b68910fa116ca6066e2f6e/pipelines/fbc-builder/fbc-builder.yaml#L102C1-L103C42)
```suggestion
  params:
    - default: "false"
    description: Enable cache proxy configuration
    name: enable-cache-proxy
```